### ISSUE TITLE
Add examples for use with d3

### ISSUE THREAD

**Issue by lucaswerkmeister** [zazuko/d3-sparql#4]
I think it’s a bit odd that the examples for a d3 support library stop at dumping the data to the console and don’t actually feed them into d3 :) an example that creates some chart would be very helpful IMHO.

**Comment by l00mi** [comment]
Yes yes! You are 100% right. I hope to find soon some time to make some nice and also insightful visualisations. (Second part is unfortunately a bit labour intensive.

Pull-requests highly encouraged!

**Comment by l00mi** [comment]
Also I like to open this Issue up for sharing ideas of visualisations based on any SPARQL endpoint.

**Comment by ktk** [comment]
See commit above to get an idea, not D3 yet but Highcharts. Data pulled from Statistik Stadt Zürich, as RDF Data Cube.

**Comment by ktk** [comment]
Whole bunch of examples by @mmznr

* https://github.com/mmznr/DINAcon/
